pimd/pim_mlag.c

```diff
@@ -1082,6 +1082,14 @@ void pim_instance_mlag_terminate(struct pim_instance *pim)
 	pim->inst_mlag_intf_cnt = 0;
 }
 
+void pim_mlag_terminate(void)
+{
+	stream_free(router->mlag_stream);
+	router->mlag_stream = NULL;
+	stream_fifo_free(router->mlag_fifo);
```

Comment:
this one isn't NULL-safe, sadly - shouldn't it be checked here?

Reply:
I don't see how we could ever get here with a non null pointer in all honesty
pim_terminate is called as we are shutting down this is the last thing we do basically